### PR TITLE
add secrets handling in kubernetes using vault blog

### PR DESCRIPTION
**Pull request by rubyrat**

### CODE REVIEW

**Comment by rubyrat** [comment]
@mvainio-verifa can you check the blog's ok, especially the code blocks (I changed one of them to hcl as per TODO comment) Thanks!

**Comment by rubyrat** [comment]
and just putting reminder to self here - to link back to this blog from first one - https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/

File: src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+- Kubernetes
+- Vault
+- Jenkins
+date: 2022-10-12
+image: "/blogs/secrets-handling-in-kubernetes-using-hashicorp-vault/secrets-handling-in-kubernetes-using-hashicorp-vault.png"
+featured: true
+
+---
+
+When running Jenkins jobs using the Kubernetes plugin, there are many ways to fetch secrets from HashiCorp Vault. [In the previous post](https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/) we stored the secrets in Kubernetes, but let’s look at options that don’t persist secrets in the cluster.
```
Comment (mvainio-verifa) [inline src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md:17]: I think we need to drop the `https://verifa.io` from the links that stay on the site. Right @jlarfors?

```
# before:
https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/
# after:
/blog/secrets-handling-in-kubernetes-a-jenkins-story/
```

File: src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+## Summary/Conclusion
+
+Looking at the pipeline, there’s actually nothing Jenkins specific to the solution presented here. We could run any pod in the `jenkins-dev` namespace with service account `jenkins-dev` to access the secrets that the policy in Vault allows. No hard coded and long-lived credentials laying around! No need for complex groovy syntax, Helm charts or Jenkins plugins. This keeps the pipeline flexible in case you (or the management) decide to ditch Jenkins and go for another tool to run your CI/CD pipelines.
+
+
+This is a follow up blog to [Secrets handling in Kubernetes - A Jenkins story](https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/), where we explore some ways of getting secrets into Jenkins which we deploy in Kubernetes, without the use of an external secrets manager.
```
Comment (mvainio-verifa) [inline src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md:267]: Here is also an internal link.

```
https://verifa.io/blog/secrets-handling-in-kubernetes-a-jenkins-story/
# to:
/blog/secrets-handling-in-kubernetes-a-jenkins-story/
```

**Review by mvainio-verifa — CHANGES_REQUESTED** [review]
Looks great in a local build and no issues with code blocks, just need to fix the internal links so clicking them doesn't open a new tab.

File: src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md (hunk 1 above)
Comment (rubyrat) [reply src/posts/secrets-handling-in-kubernetes-using-hashicorp-vault.md:17]: ah, just had a quick look into this and yes internal links shouldn't open a new tab (better for UX, SEO, etc etc). Only external links to open in a new tab.

**Pull request merged**